pager/src/main/java/com/google/accompanist/pager/PagerIndicator.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.pager
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+/**
+ * Contains default values used for [PagerIndicator].
+ */
+object PagerDefaults {
+    /**
+     * Default spacing used for [PagerIndicator].
+     */
+    val Spacing = 8.dp
+
+    /**
+     * Default indicator size used for [PagerIndicator].
+     */
+    val Size = 8.dp
+
+    /**
+     * Default shape used for [PagerIndicator].
+     */
+    val Shape = CircleShape
+
+    /**
+     * Default Indicator color value used for [PagerIndicator].
+     */
+    val IndicatorColor = Color.White
+
+    /**
+     * Default alpha value used for generating the inactive indicators color for [PagerIndicator].
+     */
+    val InactiveIndicatorColorAlpha = 0.6f
```

Comment:
Maybe use `ContentAlpha.disabled`

Reply:
Same as with the color, this is part of material wich is currently not pulled in